src/AWS.Deploy.DockerEngine/DockerEngine.cs

```diff
@@ -126,9 +126,9 @@ private ImageMapping GetImageMapping()
         {
             var content = ProjectUtilities.ReadDockerFileConfig();
             var definitions = JsonConvert.DeserializeObject<List<ImageDefinition>>(content);
-            var mappings = definitions.Where(x => x.SdkType.Equals(_project.SdkType)).FirstOrDefault();
+            var mappings = definitions.First(x => x.SdkType.Equals(_project.SdkType));
 
-            return mappings.ImageMapping.FirstOrDefault(x => x.TargetFramework.Equals(_project.TargetFramework));
+            return mappings.ImageMapping.First(x => x.TargetFramework.Equals(_project.TargetFramework));

```

Comment:
is it guaranteed to have matching element in both `definitions` and `ImagaMapping` collections?

First will crash at runtime if the element does not exit.

Reply:
Fixed